dev/src/transaction.ts

```diff
@@ -61,28 +62,17 @@ const READ_AFTER_WRITE_ERROR_MSG =
 export class Transaction {
   private _firestore: Firestore;
   private _writeBatch: WriteBatch;
-  private _requestTag: string;
+  private _requestTag?: string;
```

Comment:
It's not immediately obvious to me why this can't be passed into the constructor?

And, er, on that note, I can't figure out where this is ever assigned now?  Seems like it'll always be undefined.

Reply:
It should have been assigned in `runTransaction()`. This does lead to a bunch of `?` and `!` in the code though, so I happily moved it back to the constructor.